mv/plan/pom.xml

```diff
@@ -26,7 +26,7 @@
     <relativePath>../../pom.xml</relativePath>
   </parent>
 
-  <artifactId>carbondata-mv-plan</artifactId>
+  <artifactId>carbondata-mv-plan_${spark.binary.version}</artifactId>
```

Comment:
It is to name it `carbondata-mv-plan-spark_${spark.binary.version}` to indicate the version number is for spark

Reply:
yes, we want the user to have both 2.3 and 2.4 dependencies